source/vocab/items.ttl

```diff
@@ -26,13 +28,33 @@
 # BF2 ITEM/HOLDING LOCATION
 
 :Item a owl:Class ;
-    rdfs:label "Item"@en, "Bestånd"@sv ;
+    rdfs:label "Item"@en, "Exemplar"@sv ;
+    ptg:abstract true ;
     :category marc:hold;
     rdfs:subClassOf :Embodiment;
     skos:closeMatch rdaent:Item;
     owl:equivalentClass bf2:Item, sdo:Product, holding:Item .
     #TODO?: owl:equivalentClass [ owl:unionOf (sdo:Product sdo:Offer) ] ;
 
+:ItemHolding a owl:Class ;
+    rdfs:label "Item holding"@en, "Bestånd"@sv ;
+    owl:equivalentClass sdo:SomeProducts ;
+    rdfs:subClassOf :Item .
+
+:SomeItem a owl:Class ;
+    ptg:abstract true ;
+    rdfs:label "Some item"@en, "Något exemplar"@sv ;
+    rdfs:subClassOf :Item .
+
+:SingleItem a owl:Class ;
+    rdfs:label "Single item"@en, "Enskilt exemplar"@sv ;
+    rdfs:subClassOf :SomeItem .
+
+:itemUsed a owl:ObjectProperty ;
+    rdfs:label "Item used"@en, "Exemplaret som använts"@sv ;
+    rdfs:domain :ProvisionActivity ;
+    rdfs:range :SingleItem .
```

Comment:
```suggestion
    rdfs:range :Item .
```
Allow used items to refer to a single item, some items or an entire holding thereof.